server/config/client.go

```diff
@@ -122,9 +122,9 @@ func GenerateClientConfig(c *model.Config, telemetryID string, license *model.Li
 	props["AllowCustomThemes"] = "true"
 	props["AllowedThemes"] = ""
 	props["DataRetentionEnableMessageDeletion"] = "false"
-	props["DataRetentionMessageRetentionDays"] = "0"
+	props["DataRetentionMessageRetentionHours"] = "0"
 	props["DataRetentionEnableFileDeletion"] = "false"
-	props["DataRetentionFileRetentionDays"] = "0"
+	props["DataRetentionFileRetentionHours"] = "0"
```

Comment:
Unrelated to this PR: Why is this a string and not an int?

Reply:
I'm unsure why everything has to be string here